openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetailPage.tsx

```diff
@@ -474,89 +474,94 @@ const PoliciesDetailPage = () => {
                       Add Rule
                     </Button>
                   </Tooltip>
-                  <Space className="tw-w-full" direction="vertical" size={16}>
+
+                  <Space
+                    className="tw-w-full tw-pt-4"
+                    direction="vertical"
+                    size={20}>
                     {policy.rules.map((rule) => (
-                      <Collapse key={uniqueId()}>
-                        <Panel
-                          header={
-                            <Space
-                              className="tw-w-full"
-                              direction="vertical"
-                              size={4}>
-                              <Space
-                                align="baseline"
-                                className="tw-w-full tw-justify-between"
-                                size={4}>
-                                <Typography.Text
-                                  className="tw-font-medium tw-text-base tw-text-grey-body"
-                                  data-testid="rule-name">
-                                  {rule.name}
-                                </Typography.Text>
-                                {getRuleActionElement(rule)}
-                              </Space>
-                              <div
-                                className="tw--ml-5"
-                                data-testid="description">
+                      <Card key={rule.name || 'rule'}>
+                        <div className="tw-w-full tw-flex tw-justify-between tw-pb-7">
+                          <Typography.Text
+                            className="tw-font-medium tw-text-base tw-text-grey-body"
+                            data-testid="rule-name">
+                            {rule.name}
+                          </Typography.Text>
+                          {getRuleActionElement(rule)}
+                        </div>
```

Comment:
why do we remove space component and used div element?

Reply:
Updated and make space consistent.

Reply:
Thanks, let's wait for checks to be pass!